### PR TITLE
YOLO Training with Combined Dataset

### PR DESCRIPTION
**Pull request by audreyyluu**
Our best results:
Precision = 0.74
Recall = 0.689
mAP50 = 0.749
mAP50-95 = 0.589

Our best configuration:
epochs=80 
imgsz=480 
batch=16 
learning rate (lr0)=0.001

Part of #9

### CODE REVIEW

**Review by margoglvz — CHANGES_REQUESTED** [review]
@audreyyluu @Arapaimace 

hi guys tysm for your PR! I cant see the total results since I am assuming you guys ran it inside a google colab notebook but the results look really good! As mentioned in a previous meeting, I think for the next task of training you guys can test out different models to see if another can perform just as well or outperform yolov11s.  Also, for the results I will assume that it is YOLOv11s since that is what your workflow presentation mentions! 

I saw that you guys also got some visualizations which you put into the worfklow presentation, so if possible can you guys put the link and **share** your colab notebook so I can review that as well? thank you!!
